DapperDoodle/Case.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using System.Globalization;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace DapperDoodle
+{
+    public enum Case
+    {
+        IgnoreCase,
+        Uppercase,
+        PascalCase,
+        Lowercase,
+        CamelCase,
+        KebabCase,
+        SnakeCase
+    }
+    
+    public static class CaseHelpers
+    {
+        /// <summary>
+        /// The ConvertCase method currently only supports conversions from Pascal Case to everything else. Other support can later be added.
+        /// </summary>
+        /// <param name="value"></param>
+        /// <param name="casing"></param>
+        /// <returns></returns>
+        /// <exception cref="ArgumentNullException"></exception>
+        public static string ConvertCase(this string value, Case casing)
+        {
+            switch (casing)
```

Comment:
minor: logic here is getting long (this is the curse of `switch` statements). Suggest strategy pattern.

Reply:
Thanks. I will definitely have a look at the strategy pattern. 🙂